pkg/accounting/data.go

```diff
@@ -2,16 +2,18 @@
 package accounting
 
 var activeSeriesByCheckType = map[string]int{
-	"dns":            84,
-	"dns_basic":      28,
-	"http":           118,
-	"http_basic":     34,
-	"http_ssl":       122,
-	"http_ssl_basic": 38,
-	"ping":           82,
-	"ping_basic":     26,
-	"tcp":            37,
-	"tcp_basic":      23,
-	"tcp_ssl":        41,
-	"tcp_ssl_basic":  27,
+	"dns":              84,
+	"dns_basic":        28,
+	"http":             118,
+	"http_basic":       34,
+	"http_ssl":         122,
+	"http_ssl_basic":   38,
+	"ping":             82,
+	"ping_basic":       26,
+	"tcp":              37,
+	"tcp_basic":        23,
+	"tcp_ssl":          41,
+	"tcp_ssl_basic":    27,
+	"traceroute":       20,
+	"traceroute_basic": 20,
```

Comment:
Does this make sense? If these are the same, it feels to me like we don't actually have a "basic" version of this check... 🤔

Reply:
I think the basic version is the one we have, probably. I'm thinking we just don't have a "basic" toggle/setting for v1, and as we come up with more use cases that can be supported by more metrics we can add it in